src/status_im/ui/screens/desktop/main/add_new/views.cljs

```diff
@@ -67,29 +80,32 @@
         (i18n/label :new-public-group-chat)]]
       [react/text {:style styles/new-contact-subtitle} (i18n/label :public-group-topic)]
       [react/view {:style styles/new-contact-separator}]
-      [react/view {:style styles/add-contact-edit-view}
-       [react/view {:style {:flex 1}}
-        [react/text-input {:flex             1
-                           :ref              #(when (and (nil? @topic-input-ref) %)
-                                                (.setNativeProps % (js-obj "text" "#"))
-                                                (reset! topic-input-ref %))
-                           :style            styles/add-contact-input
-                           :font             :default
-                           :selection-color  colors/hawkes-blue
-                           :on-change        (fn [e]
-                                               (let [native-event (.-nativeEvent e)
-                                                     text (.-text native-event)
-                                                     [_ before after] (first (re-seq #"(.*)\#(.*)" text))]
-                                                 (.setNativeProps @topic-input-ref (js-obj "text" (str "#" before after)))
-                                                 (re-frame/dispatch [:set :public-group-topic (subs text 1)])))}]]
-       [react/touchable-highlight {:disabled topic-error
-                                   :on-press #(when-not topic-error
-                                                (do
+      (let [disable? (or (not (string/blank? topic-error))
+                         (string/blank? topic))]
+        [react/view {:style styles/add-contact-edit-view}
+         [react/view {:flex 1}
```

Comment:
Are you sure it actually has any effect? I found that some style props will be applied only if passed in `styles`.

Reply:
Yes, I specifically checked that `{:flex 1}` does what I need. Will have to dig to the root cause of this `{:style ...}` problem at some point.